server/storage/storage_impl.go

```diff
@@ -345,26 +303,30 @@ func (s *metaStorageImpl) CreateTable(ctx context.Context, clusterID uint32, sch
 }
 
 func (s *metaStorageImpl) GetTable(ctx context.Context, clusterID uint32, schemaID uint32, tableName string) (*clusterpb.Table, bool, error) {
-	value, err := s.Get(ctx, makeNameToIDKey(clusterID, schemaID, tableName))
+	resp, err := s.client.Get(ctx, makeNameToIDKey(s.rootPath, clusterID, schemaID, tableName))
```

Comment:
Maybe we should provide a method called `getValue` in the etcdutil package to avoid the duplicate codes.